src/test/java/io/github/linuxforhealth/FHIRConverterTest.java

```diff
@@ -154,6 +151,40 @@ public void test_ORU_r01_without_status() throws IOException {
     assertThat(status).isEqualTo(DiagnosticReport.DiagnosticReportStatus.UNKNOWN);
   }
 
+  @Test
+  public void test_dosage_output() throws  IOException {
+    String hl7message =
+            "MSH|^~\\&|MyEMR|DE-000001| |CAIRLO|20160701123030-0700||VXU^V04^VXU_V04|CA0001|P|2.6|||ER|AL|||||Z22^CDCPHINVS|DE-000001\r" +
+                    "PID|1||PA123456^^^MYEMR^MR||JONES^GEORGE^M^JR^^^L|MILLER^MARTHA^G^^^^M|20140227|M||2106-3^WHITE^CDCREC|1234 W FIRST ST^^BEVERLY HILLS^CA^90210^^H||^PRN^PH^^^555^5555555||ENG^English^HL70296|||||||2186-5^ not Hispanic or Latino^CDCREC||Y|2\r" +
+                    "PD1|||||||||||02^REMINDER/RECALL - ANY METHOD^HL70215|N|20140730|||A|20140730|\r" +
```

Comment:
Suggest just including the segments needed for this function.  Not sure what the minimum is but for sure do not need PD1, NK1, OBX.  Think RXR could also be removed.